js/client-chat.js

```diff
@@ -857,6 +857,13 @@
 				}
 				return text; // Send the /avatar command through to the server.
 
+			case 'reply':
```

Comment:
The replacement should be immediate upon typing `/r ` or `/reply `, it shouldn't wait until you press Enter.